client/static/js/hatohol_monitoring_view.js

```diff
@@ -19,6 +19,7 @@
 
 window.onerror = function(errorMsg, fileName, lineNumber) {
   var place = "[" + fileName + ":" + lineNumber + "]";
+  console.error(place + " " + errorMsg);
```

Comment:
It works with IE11 on my environment.
But I don't test it with IE10 or older yet.